cpp/src/codegen/llvm_generator_test.cc

```diff
@@ -42,6 +42,11 @@ TEST_F(TestLLVMGenerator, VerifyPCFunctions) {
 
   llvm::Module *module = generator->module();
   for (auto &iter : registry_) {
+    if (iter.needs_holder()) {
+      // TODO : need a way to verify these too.
```

Comment:
please add a jira (if not planning to address immediately)
would need this if we move the extract functions en masse..

Reply:
i think i found a way to fix this. will do it as a different PR in the same jira after fixing the java linker issue.